src/Hyprpaper.hpp

```diff
@@ -34,6 +34,7 @@ class CHyprpaper {
     std::vector<std::unique_ptr<SMonitor>> m_vMonitors;
 
     bool        m_bIPCEnabled = true;
+    std::string explicitConfigPath;
```

Comment:
you saw the Hungarian naming convention above and you decided to ignore it

Reply:
I can change that, but that's [exactly how it is](https://github.com/hyprwm/Hyprland/blob/main/src/Compositor.hpp?plain=1#L169) (currently) in hyprland codebase, which also uses hungarian naming convention - thought you named it like that to allude to it being set directly without setter/getter.

Reply:
no, it was fufexan and he is a goof

Reply:
Changed.